.github/workflows/CI.yml

```diff
@@ -55,8 +55,11 @@ jobs:
       if: ${{ matrix.connector == 'turbodbc' }}
 
     - name: Checkout test environment
-      run: git clone https://github.com/exasol/integration-test-docker-environment.git
+      run: git clone --depth 1 --branch ${ITDE_TAG} ${ITDE_URL}
```

Comment:
nice, in general, I currently suggest to add a shell script which starts the itde, you can use the same for local setuo

Reply:
like here https://github.com/exasol/advanced-analytics-framework/blob/main/scripts/start_integration_test_environment.sh

Reply:
> nice, in general, I currently suggest to add a shell script which starts the itde, you can use the same for local setup

nox currently starts the itde not sure though if it also should take care of the version of the dependency. I would assume the default for the local dev machine is main anyway and for the CI it should be pinned.

Or is there critical issue with this in your opinion?

Reply:
Ah you are right there was an automation. In general, I would suggest to pin the version, to decouple things